<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Toto Master</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="packetButton">
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>490</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать пакет</string>
    </property>
   </widget>
   <widget class="QTableView" name="predictionTableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>371</width>
      <height>481</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="countEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Хозяева</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Мин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Макс</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minhEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxhEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="mindEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxdEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Мин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Макс</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ничья</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mingEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxgEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Мин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Макс</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Гости</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minfEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxfEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>240</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Мин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Макс</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Фавориты</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minnEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxnEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>300</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Мин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>310</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Макс</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Нейтрал</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minoEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxoEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>360</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>370</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Мин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Макс</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Аутсайдеры</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savePredictionButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>410</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить прогноз</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadPredictionButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>450</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить прогноз</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>531</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество купонов</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>20</width>
      <height>551</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTableView" name="resultTableView">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>111</width>
      <height>481</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>10</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="loadPacketButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить пакет</string>
    </property>
   </widget>
   <widget class="QTableView" name="calculationTableView">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>0</y>
      <width>256</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>500</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Суммарный выигрыш</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalWonEdit">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>490</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="countComboBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во</string>
    </property>
   </widget>
   <widget class="QPushButton" name="impotButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>исключить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="importCountEdit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>490</y>
      <width>91</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalSpetEdit">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>530</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>540</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Вложено</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalProfitEdit">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>570</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>580</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Прибыль</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>predictionTableView</tabstop>
  <tabstop>minhEdit</tabstop>
  <tabstop>maxhEdit</tabstop>
  <tabstop>mindEdit</tabstop>
  <tabstop>maxdEdit</tabstop>
  <tabstop>mingEdit</tabstop>
  <tabstop>maxgEdit</tabstop>
  <tabstop>minfEdit</tabstop>
  <tabstop>maxfEdit</tabstop>
  <tabstop>minnEdit</tabstop>
  <tabstop>maxnEdit</tabstop>
  <tabstop>minoEdit</tabstop>
  <tabstop>maxoEdit</tabstop>
  <tabstop>countEdit</tabstop>
  <tabstop>packetButton</tabstop>
  <tabstop>savePredictionButton</tabstop>
  <tabstop>loadPredictionButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
